Description: modernize the AM_INIT_AUTOMAKE call in the configure script
Author: Jakub Wilk <jwilk@debian.org>
Forwarded: no
Last-Update: 2014-01-28

--- a/configure.ac
+++ b/configure.ac
@@ -1,15 +1,12 @@
-dnl Get things going...
-AC_INIT(rc.h)
+dnl Automake stuff.
+AC_INIT([rc], [1.7.1])
+AC_CONFIG_SRCDIR([rc.h])
+AM_INIT_AUTOMAKE
+
 
 RELDATE=`cat $srcdir/RELDATE`
 AC_DEFINE_UNQUOTED(RELDATE, "$RELDATE", [Release date])
 
-dnl Automake stuff.
-dnl Use this one for snapshots...
-dnl AM_INIT_AUTOMAKE(rc, 1.7s`echo $RELDATE |sed 's/-//g'`)
-dnl ...and this one for releases
-AM_INIT_AUTOMAKE(rc, 1.7.1)
-
 AM_CONFIG_HEADER(config.h)
 
 dnl Find a standard C compiler
